examples/widget/db/README.md

```diff
@@ -0,0 +1,32 @@
+# Database
+
+This example uses the [Prisma](https://www.prisma.io/) ORM on top of a [PostgreSQL](https://www.postgresql.org/) database.
```

Comment:
@igneel64 Will you ping the Prisma team for that? Maybe we can get some backlinks.